Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/list/VaccinationListViewModel.kt

```diff
@@ -125,6 +144,16 @@ class VaccinationListViewModel @AssistedInject constructor(
         events.postValue(Event.NavigateToVaccinationQrCodeScanScreen)
     }
 
+    fun deleteVaccination(vaccinationCertificateId: String) {
+        launch(scope = appScope) {
+            try {
+                vaccinationRepository.deleteVaccinationCertificate(vaccinationCertificateId)
+            } catch (exception: Exception) {
```

Comment:
should we show the error to the user here ? `errors.postValue(exception)`?

Reply:
Look for `error.toErrorDialogBuilder(requireContext()).show()`

Reply:
Thanks, the error is now shown to the user.